lib/omnibus/packagers/msi.rb

```diff
@@ -26,7 +26,7 @@ class Packager::MSI < Packager::WindowsBase
       if bundle_msi
         helper_tmp_dir = Dir.mktmpdir
         parameters.store('HelperDir', helper_tmp_dir)
-        FileUtils.mv "#{install_dir}/bin/upgrade-helper.exe", "#{helper_tmp_dir}"
+        fileutils.mv "#{install_dir}/bin/upgrade-helper.exe", "#{helper_tmp_dir}"
```

Comment:
For instance: https://github.com/DataDog/omnibus-ruby/blob/datadog-5.5.0/lib/omnibus/builder.rb#L616